content/articles/working-with-github-web-api-in-python/index.md

```diff
@@ -0,0 +1,241 @@
+### Introduction
+This tutorial will teach you how to create a self-contained application that generates a visualization based on the information it obtains. Instead of requesting complete pages, a web application programming interface will be used by your program (API) to get certain information from the web. We'll use data from [GitHub](https://github.com/), a website where programmers can contribute to projects. We'll request information concerning Python projects on GitHub using Github's API.
+
+### Prerequisites
+You'll need some Python skills to continue with this tutorial.
+
+### What you will learn
+- Using an API call to request data
+- Requests installation
+- Keeping track of an API response
+- Using the response dictionary
+- Summing up the top repositories
+
+### Requesting data using an API call 
+GitHub's web API allows you to make API requests for a range of data. Type the following into your web browser URL bar and press **enter** to see how an API call appears like:
+
```

Comment:
Add the goal of this tutorial. 

What is the objective of this code? What are you trying to explain to the reader?